pinot-spi/src/main/java/org/apache/pinot/spi/utils/ArrayCopyUtils.java

```diff
@@ -239,4 +278,259 @@ public static void copy(byte[][] src, BigDecimal[] dest, int length) {
       dest[i] = BigDecimalUtils.deserialize(src[i]);
     }
   }
+  public static void copy(int[][] src, int[][] dest, int length) {
```

Comment:
Is it possible to do this refactoring in a separate PR unless it is absolutely needed ?

Reply:
Done - reverted refactoring in BaseTransformFunction. Will put up a separate PR.